src/zwreec/backend/zcode/zfile.rs

```diff
@@ -677,13 +681,26 @@ impl Zfile {
         self.data.append_byte(variable);
     }
 
-    /// prints the value of a variable (only ints a possibe)
+    /// Prints the value of a variable (only ints a possibe)
```

Comment:
Ich finde es schon lustig, dass hier ein Tippfehler korrigiert wurde, aber der zweite Tippfehler in der Zeile nicht :D